modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointment.rb

```diff
@@ -245,8 +241,6 @@ def proposed_times
         end
 
         def status
-          return STATUSES[:hidden] if HIDDEN_STATUS.include?(appointment[:status])
```

Comment:
I'm sure this is a leftover from some previous design, but it's morphed into something unnecessarily indirect.